samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManagerFactory.scala

```diff
@@ -62,16 +64,23 @@ class KafkaCheckpointManagerFactory extends CheckpointManagerFactory with Loggin
     val jobName = config.getName.getOrElse(throw new SamzaException("Missing job name in configs"))
     val jobId = config.getJobId.getOrElse("1")
 
-    val systemName = config
-      .getCheckpointSystem
-      .getOrElse(throw new SamzaException("no system defined for Kafka's checkpoint manager."))
+    val (systemName: String, systemFactory : SystemFactory) =  org.apache.samza.util.Util.getCheckpointSystemStreamAndFactory(config)
```

Comment:
Can't we import `Util` here instead of a fully qualified name here?

Reply:
I don't think this should be a util method, can we move that method here?

Reply:
well, I put it in Utils because its Coordinator counterpart was there, but, I guess, it makes more sense for it to be in this class, so I will move and refactor it.